src/itemset.js

```diff
@@ -181,7 +213,7 @@ define([
             deserialize: function (data, key, mug) {
                 _.each(data[key], function (value, i) {
                     var children = mug.form.getChildren(mug),
-                        itemset = children[i] || afterDynamicSelectInsert(mug.form, mug),
+                        itemset = children[i] || afterDynamicSelectInsert(mug.form, mug, _.first(data.itemsetData)),
```

Comment:
This should use `value` instead of `_.first(data.itemsetData)`. However, I think the modification to `afterDynamicSelectInsert` (passing a third parameter) should not be necessary. Populating the `itemset` mug should be done by [line 219](https://github.com/dimagi/Vellum/pull/636/files/a4aec3fbb032e3265b241c0cc04a059664cf3194#diff-f15dee15f58a804357f7fca0280de97dR219) below. I'm guessing the bug is in `Itemset.spec.itemsetData.deserialize`.